package.json

```diff
@@ -47,8 +47,8 @@
     "@koa/cors": "^5.0.0",
     "@koa/router": "^12.0.0",
     "@opentelemetry/api": "^1.4.1",
-    "@opentelemetry/exporter-logs-otlp-http": "0.41.2",
-    "@opentelemetry/exporter-trace-otlp-http": "^0.41.2",
+    "@opentelemetry/exporter-logs-otlp-proto": "0.46.0",
+    "@opentelemetry/exporter-trace-otlp-proto": "0.46.0",
```

Comment:
Use the latest version and lock it. These are experimental packages and breaking changes happen often.